src/pe_reports/__init__.py

```diff
@@ -1,9 +1,8 @@
-"""The example library."""
+"""The pe_reports library."""
 # We disable a Flake8 check for "Module imported but unused (F401)" here because
 # although this import is not directly used, it populates the value
 # package_name.__version__, which is used to get version information about this
 # Python package.
-from ._version import __version__  # noqa: F401
-from .example import example_div
-
-__all__ = ["example_div"]
+# Third-Party Libraries
+from _version import __version__  # noqa: F401
+from report_generator import main  # noqa: F401
```

Comment:
This needs to be a relative import. Additionally why are you importing this and not adding it to `__all__`? The above case with `__version__` is a special case. This would remove the need for the `noqa` for this import as well.

```suggestion
from .report_generator import main  # noqa: F401
```

Reply:
Resolved in 44ee65c